src/IntegrationUtils.ts

```diff
@@ -215,7 +278,11 @@ function createNewIntegration(integrationFileUri: vscode.Uri, devMode? : boolean
 				});
 			}			
 		}
-		commandString += ' "' + absoluteRoot + '"';
+		if (propertyArray && propertyArray.length > 0) {
+			propertyArray.forEach(prop => {
+				commandString += ` -p ${prop}`;
+			});
+		}			
 		console.log(`commandString = ${commandString}`);
```

Comment:
do we need to log this?

Reply:
I find it useful. The user never sees it, but it's been extremely helpful during debugging